lib/install/phases/bootstrap.go

```diff
@@ -197,6 +198,7 @@ func (p *bootstrapExecutor) configureSystemDirectories() error {
 		filepath.Join(stateDir, "planet", "etcd"),
 		filepath.Join(stateDir, "planet", "registry"),
 		filepath.Join(stateDir, "planet", "docker"),
+		filepath.Join(stateDir, "planet", "kubelet"),
```

Comment:
Needed to have additional directories explicitly created in order to correctly label them (see also for `logrange` below).